build-system/externs/amp.extern.js

```diff
@@ -624,7 +624,7 @@ AmpElement.prototype.collapse = function () {};
 let Signals = class {};
 /**
  * @param {string} unusedName
- * @return {number|!Error|null}
+ * @return {?number|?Error}
```

Comment:
primitives can never be nullable
```suggestion
 * @return {number|?Error}
```